it/controllers/errors/AgentAuthErrorControllerISpec.scala

```diff
@@ -50,7 +50,7 @@ class AgentAuthErrorControllerISpec extends IntegrationTest with ViewHelpers {
         titleCheck(ExpectedResults.title)
         h1Check(ExpectedResults.heading,"xl")
         textOnPageCheck(ExpectedResults.youCan, Selectors.youCan)
-        textOnPageCheck(ExpectedResults.tryAnother, Selectors.tryAnother)
+        buttonCheck(ExpectedResults.tryAnother, Selectors.tryAnother, Some(appConfig.viewAndChangeEnterUtrUrl))
```

Comment:
Should the url were checking be in its own variable, i.e. val changeUrl = "report-quarterly/income-and-expenses/view/agents/client-utr", because with the view and the test both pulling from the same app config place, if there was an issue with what we have in there, or if it was changed for whatever reason, we would never know. Same for all places youve done the url check.

Reply:
Changed to this